### PR TITLE
Replaced android.app.AlertDialog with android.support.v7.app.AlertDialog in ShowQrActivity

### LINKED ISSUE #744: Use android.support.v7.app.AlertDialog instead of android.app.AlertDialog in ShowQrActivity
**Summary:**
The support library version provides a fallback for devices using an older version of android.

### PR DESCRIPTION
**Pull request by dubeyaayush07**
Fixes #744

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by naman14** [comment]
@dubeyaayush07 Can you also do the same in DialogBox and SettingsActivity? They are also using `android.app.AlertDialog`

**Comment by dubeyaayush07** [comment]
@naman14 done

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/DialogBox.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package org.mifos.mobilewallet.mifospay.utils;
 
-import android.app.AlertDialog;
+import android.support.v7.app.AlertDialog;;
```
Comment (naman14) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/DialogBox.java]: extra semicolon

Comment (dubeyaayush07) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/DialogBox.java]: Corrected

**Pull request merged**